src/main/java/task/Task.java

```diff
@@ -0,0 +1,87 @@
+package task;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * This class encapsulates a Task.
+ */
+public class Task {
+    private String description;
+    private boolean status;
```

Comment:
Perhaps can name this boolean sounds more like a boolean?

Reply:
Oops! Thanks for catching that! Will update on my next commit!